### ISSUE TITLE
Shellschock.io is not working 

### ISSUE THREAD

**Issue by AbisGameryt** [nfriedly/node-unblocker#133]
It does not connect to servers.

**Comment by FeaturedSpace** [comment]
It is likely that socket communication is not being properly processed. This is an advanced feature and I'm not sure if it's been fully implemented/explored yet. As of right now, I would not expect a JS-based unblocker to run IO games.

**Comment by nfriedly** [comment]
I feel like I might be missing something, but it doesn't seem to have any actual game at shellshocker.io - just reviews of a few games (?)

**Comment by FeaturedSpace** [comment]
Ah I just assumed it was a game 😅

**Comment by AbisGameryt** [comment]
Oops sorry its https://shellshock.io/ and I am guessing its because if the websoket. I am the only one in my high School that knows programming and i wanted to find a way to unblock stuff but everyone wants https://shellshock.io/  but As you siad a couple of days ago it is possible but very hard.

**Comment by nfriedly** [comment]
aah, I see. Fun game, but yea, it uses websockets, so not currently compatible with the proxy.

**Comment by RoylatGnail** [comment]
Should we close this or merge into #118?

**Comment by nfriedly** [comment]
Yea, I suppose so.

**Comment by easella** [comment]
alloyproxy works with shellshockers: [click here](https://partialvirtualredundancy-1.awdrgyjil1234.repl.co/)
